drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/client_export_uploader.rb

```diff
@@ -45,11 +47,15 @@ def run!
     end
 
     def self.can_run?
-      !!new.send(:credentials)
+      !!new(pre_zipped_data: 'nothing').send(:credentials)
```

Comment:
Unrelated but, on line 73, we should use scope `active` on the remote credential.
Would be nice if the slug was named something more generic like "ac_data_warehouse_sftp_server" or something, but its probably not worth changing.

Reply:
That slug is only in a few places, so I can just change it.